
class FOG_Exception(Exception):
    pass

class FOG_AlreadyGotOneError(FOG_Exception):
    pass
